components/next_steps_view/steps.ts

```diff
@@ -135,11 +139,25 @@ export const Steps: StepType[] = [
         roles: [],
         component: DownloadAppsStep,
         visible: true,
-        finishButtonText: {
+        completeStepButtonText: {
             id: t('next_steps_view.next'),
             defaultMessage: 'Next step',
         },
     },
+    {
+        id: RecommendedNextSteps.CREATE_FIRST_CHANNEL,
+        title: {
+            titleId: t('next_steps_view.titles.createChannel'),
+            titleMessage: 'Create your first channel',
+        },
+        roles: ['system_admin'],
+        component: CreateFirstChannelStep,
+        visible: true,
```

Comment:
We could check the feature flag here directly to hide the step, instead of harcoding to `true` and then filtering later.

Reply:
because the way the steps list array and the steps component is implemented, this is not easy. The information about the feature flag doesn't seem to load on the first render, and I can not use any function like the useSelector to force a component rerender after this value get's loaded into the state. This steps.ts file doesn't have neither a stateToProps nor it is a FC where I can use hooks. So, to do this, the entire component would need refactoring that is not worth the effort for this A/B test and is risky in terms of adding some other regressions.